Resources/Prototypes/Reagents/Materials/metals.yml

```diff
@@ -3,7 +3,7 @@
   stackEntity: SheetSteel1
   name: materials-steel
   icon: { sprite: Objects/Materials/Sheets/metal.rsi, state: steel }
-  price: 0.05
```

Comment:
As above please explain why when changing balance.

Reply:
The price of steel is inexplicably low compared to the purchase price. But yes, now that I look at it, I don't understand why the price has changed so little. Maybe I wasn't sure it would be accepted.
Back to the previous price level.